.github/workflows/ci.yml

```diff
@@ -22,6 +22,7 @@ jobs:
           [
             "django32-drflatest",
             "django40-drflatest",
+            "django42-drflatest",
```

Comment:
also update `translations` check to run with both `django32` and `django42`. 
e.g. 
```
translations-django32,
translations-django42
```

Reply:
done with comment.